src/lib/es2020.intl.d.ts

```diff
@@ -98,7 +97,39 @@ declare namespace Intl {
         unit?: RelativeTimeFormatUnit;
     }
 
-    interface RelativeTimeFormat {
+    class RelativeTimeFormat {

```

Comment:
This change means you can't merge another declaration of RelativeTimeFormat to type expando assignments to it. The current version already forbids calls with the error "Value of type { new .... } isn't callable. Did you mean to include 'new'?". We should stay with the old interface+const.

Reply:
gotcha fixed